builtin/logical/pki/path_manage_keys.go

```diff
@@ -0,0 +1,195 @@
+package pki
+
+import (
+	"context"
+	"strings"
+
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/helper/certutil"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+func pathGenerateKey(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "keys/generate/(internal|exported|kms)",
+
+		Fields: map[string]*framework.FieldSchema{
+			keyNameParam: {
+				Type:        framework.TypeString,
+				Description: "Optional name to be used for this key",
+			},
+			keyTypeParam: {
+				Type:        framework.TypeString,
+				Default:     "rsa",
+				Description: `Type of the secret key to generate`,
+			},
+			keyBitsParam: {
+				Type:        framework.TypeInt,
+				Default:     2048,
+				Description: `Type of the secret key to generate`,
```

Comment:
I don't think this description is correct. :)

Reply:
Addressed in #15319.